external/wakaama/examples/client/object_connectivity_stat.c

```diff
@@ -56,35 +57,240 @@ typedef struct
     int64_t   avrMessageSize;
     int64_t   messageCount;           // private for incremental average calc.
     bool      collectDataStarted;
+	uint32_t collectionPeriod;
 } conn_s_data_t;
 
+
+/****************************************************************************
+ * Name: fetch_smsTx
+ *
+ * Description:
+ *   To get the smsTx statistic value for lwm2m from connectivity manager
+ *
+ * Returned Value:
+ *   
+ *
+ ****************************************************************************/
+ void fetch_smsTx(int *value);
```

Comment:
All functions are already removed with this commit.
https://github.com/Samsung/TizenRT/pull/250/commits/68a94144f84875c7c1d86ec7e451baedb2a54d37

Thanks.